apps/data/io/cgd.py

```diff
@@ -83,16 +86,36 @@ def _to_entries(self, ignore_aliases):
                 aliases = feature['aliases'].split('|')
 
             for identifier in (feature['name'], *aliases):
+
+                # The Entry primary key is precomputed given an identifier and
+                # a repository. By doing so, we will be able to create a set()
+                # to remove duplicates eventually generated by aliases.
+                pk = blake2b(
+                    bytes(
+                        '/'.join((
```

Comment:
why not on a single line?

Reply:
At first there was one more argument. But now you are right.